README.md

```diff
@@ -1,101 +1,96 @@
+For realtime discussion please see the pymavlink [Gitter channel](https://gitter.im/ArduPilot/pymavlink)
 
 Examples can be found [in the repository](examples/) or in the [ArduSub book](https://www.ardusub.com/developers/pymavlink.html)
 
 
 # Installation 
 
-Pymavlink supports both python2 and python3.
+Pymavlink supports both Python 2 and Python 3.
 
-The following instructions assume you are using Python 2 and a Debian-based (like Ubuntu) installation.
+The following instructions assume you are using Python 3 and a Debian-based (like Ubuntu) installation.
 
 ## Dependencies
 
 Pymavlink has several dependencies :
 
-    - [future](http://python-future.org/) : for python 2 and python 3 interoperability 
+    - [future](http://python-future.org/) : for Python 2 and Python 3 interoperability 
     - [lxml](http://lxml.de/installation.html) : for checking and parsing xml file 
-    - python-dev : for mavnative
+    - python3-dev : for mavnative (or python-dev for Python 2)
     - a C compiler : for mavnative
 
 Optional :
-
     - numpy : for FFT
     - pytest : for tests
 
-### On linux
+### On Linux
 
 lxml has some additional dependencies that can be installed with your package manager (here with `apt-get`) :
 
 ```bash
-sudo apt-get install gcc python-dev libxml2-dev libxslt-dev
+sudo apt-get install gcc python3-dev libxml2-dev libxslt-dev
 ```
 
 Optional for FFT scripts and tests:
 
 ```bash
-sudo apt-get install python-numpy python-pytest
+sudo apt-get install python3-numpy python3-pytest
 ```
 
 Using pip you can install the required dependencies for pymavlink :
 
 ```bash
-sudo pip2 install -U future lxml
+sudo python3 -m pip install --upgrade future lxml
 ```
 
-The -U parameter allows updating future and lxml version if it is already installed.
-
 ### On Windows
 
-Use pip to install future as for linux.
-Lxml can be installed with a windows installer from here : https://pypi.python.org/pypi/lxml/3.6.0
+Use pip to install future as for Linux.
+Lxml can be installed with a Windows installer from here : https://pypi.org/project/lxml
 
 
 ## Installation
 
 ### For users
 
-It is recommended to install pymavlink from PyPi with pip, that way dependencies should be auto install by pip.
+It is recommended to install pymavlink from PyPI with pip, that way dependencies should be auto installed by pip.
 
 ```bash
-sudo pip2 install -U pymavlink
+sudo python3 -m pip install --upgrade pymavlink
 ```
 
-The -U parameter allow to update pymavlink version if it is already installed.
-
 #### Mavnative
 
 By default, pymavlink will try to compile and install mavnative which is a C extension for parsing mavlink. Mavnative only supports mavlink1.
 To skip mavnative installation and reduce dependencies like `gcc` and `python-dev`, you can pass `DISABLE_MAVNATIVE=True` environment variable to the installation command:
 
 ```bash
-sudo DISABLE_MAVNATIVE=True pip2 install -U pymavlink
+sudo DISABLE_MAVNATIVE=True python3 -m pip install --upgrade pymavlink
```

Comment:
Are there systems out there where just `python` shouldn't work?  Do we need to specify a Python here?

Reply:
[PEP020](https://www.python.org/dev/peps/pep-0020): _Explicit is better than implicit..._

Also, https://snarky.ca/why-you-should-use-python-m-pip

On my Mac:
% `python --version ` # Python 2.7.16
% `python2 --version ` # Python 2.7.16
% `python3 --version ` # Python 3.8.5

We don't really want to encourage the use of legacy Python if we don't have to.

Reply:
on ubuntu 20.04 by default python doesn't exist ... only python3

Reply:
Those changes were reverted.